README.md

```diff
@@ -58,61 +58,110 @@ can see the routes with ```rake routes```;
 # Preparing a Development Environment
 
 ## Prerequisites
+---
```

Comment:
These `<hr>`s seem a little out of place to me. What's the thought process behind them?